lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -15,67 +13,42 @@ class FeatureCard extends React.Component {
   constructor(props) {
     super(props);
 
-    // These variables shouldn't force a state change & subsequent render
-    this.requests = [];
-    this.campaigns = [];
-    this.campaignsToExclude = setting('dsUser.activity.signups', []);
-    this.campaignsApiPage = 1;
-    this.apiClient = new ApiClient('v1');
-
     this.state = {
       actionDisabled: false,
       campaignIndex: -1,
       isSignedUp: false,
       isLoading: false,
     };
 
+    this.campaigns = [];
+    this.controller = new GalleryController();
```

Comment:
Ideally, this campaign class is a "store" that's responsible for loading in data from the API as necessary, and then "popping" out individual campaigns to the view so they can be displayed by one of the feature cards. We're not really seeing a lot of the benefit of extracting this if they all still naively fetch & maintain their own independent sets of data.